### DISCUSSION TITLE
请帮帮我！

### DISCUSSION THREAD

**Discussion by Doubudou45** [share121/inter-knot#5211] [委托]
![Screenshot_20240916_005900](https://github.com/user-attachments/assets/983add1a-170e-4533-ad18-e466e52304ec)
打不过，有什么思路吗？

**Comment by Loxia1121** [comment]
哥们你已经很强了（）用的什么队伍？

**Reply by Doubudou45** [reply]
我是用青衣 妮可 朱鸢
和艾莲 珂蕾妲 苍角

**Comment by YeYingMakas** [comment]
看看队伍嘛

**Comment by Doubudou45** [comment]
我是青衣 妮可 朱鸢
 和艾莲 珂蕾妲 苍角

**Reply by YeYingMakas** [reply]
诶，跟我配队差不多，可能单纯练度的问题🤔或者设备不流畅导致的

**Reply by Doubudou45** [reply]
是我的操作不行(╥ω╥`)

**Comment by fuqikilroy** [comment]
也许你可以先打完蓝色的，反正没有时间限制，一个青衣打七八分钟也就单通了

**Comment by Avyere** [comment]
可以用猫派露+朱青妮，我这两次都是这俩个队打的全S

**Comment by WuxuRiver** [comment]
你这个配队，嗯，技术是硬伤，除非你角色命座专武啥的都有

**Reply by Doubudou45** [reply]
艾莲和青衣都有专武，主要还是技术不行Σ(|||▽||| )

**Reply by Avyere** [reply]
推荐你派派呀，派派无论是特殊强化技还是强化技都是有无敌帧和霸体的，而且派派闪避反击的无敌帧体感比很多角色长很多

**Comment by yansai0v0** [comment]
练
